.vib/prestashop/goss/goss.yaml

```diff
@@ -0,0 +1,12 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../prestashop/goss/prestashop.yaml: {}
+  # Load scripts from .vib/common/goss/templates
```

Comment:
A new template ([check-apache-libphp.yaml](https://github.com/bitnami/containers/blob/main/.vib/common/goss/templates/check-apache-libphp.yaml)) has been recently added that I think would perfectly fit here. Could you please add it?